src/sections/Sponsors.tsx

```diff
@@ -23,11 +22,15 @@ const Sponsors = () => {
                 spacing={{base: 8, md: 14}}
                 pt={{base: 20, md: 28}}>
                 <Heading className="heading">Meet Our Sponsors</Heading>
-                <Text align='center' paddingBottom='14'>Interested in sponsoring us? Get in touch with our team at&nbsp;
+                <Text align='center' >Interested in sponsoring us? Get in touch with our team at&nbsp;
                     <Link textDecor={'underline'} href="mailto:sponsor@utmist.org">
                         sponsor@utmist.org
-                    </Link>
+                    </Link>&nbsp;and view our sponsorship package&nbsp;
+                    <Link textDecor={'underline'} href="/Package.pdf">
```

Comment:
Mostly LGTM but a few points:
1. Can you make it open in a new tab? `target="_blank"` should do the trick
2. Minor nit, but could you get the team that produced this report to remove draft from the document title? It looks a bit sus <img width="208" alt="image" src="https://user-images.githubusercontent.com/17518332/211377532-4a7a465a-a6a0-4b71-9c4a-f04cbace8a30.png">
3. Perhaps rename `Package.pdf` to `hack_the_mist_sponsorship_package.pdf` or something, so that when a sponsor downloads it it's easy to locate.